app/main.py

```diff
@@ -2,0 +1,16 @@
+from app.errors import (VaccineError, NotWearingMaskError)
+from app.cafe import Cafe
+
+
+def go_to_cafe(friends: list[dict], cafe: Cafe) -> str:
+    masks_to_buy = 0
+    for visitor in friends:
+        try:
+            cafe.visit_cafe(visitor)
+        except VaccineError:
+            return "All friends should be vaccinated"
+        except NotWearingMaskError:
+            masks_to_buy += 1
+    if masks_to_buy:
+        return f"Friends should buy {masks_to_buy} masks"
+    return f"Friends can go to {cafe.name}"
```

Comment:
You can avoid using the `if` statement here by returning the message 'Friends should buy {masks_to_buy} masks' directly. If `masks_to_buy` is 0, it means that no masks need to be bought, and the message will still make sense. This will make your code more concise and easier to read.